src/components/header/TopHeader.jsx

```diff
@@ -1,23 +1,38 @@
-import React from "react";
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Join from './../../pages/join/Join';
+import Login from './../../pages/login/Login';
 
 const TopHeader = () => {
   return (
-    <div className="w-full border-b border-[#CBCBCB] h-[47px]  top-0 flex justify-center bg-white">
-      <div className="flex min-w-[1280px] justify-end  items-center ">
-        <ul className="flex items-center  gap-[40px]">
-          <li className="relative after:content-['|'] after:absolute after:right-[-20px]">
-            <a href="#">회원가입</a>
+    <div className="top-0 flex h-[47px] w-full justify-center border-b border-[#CBCBCB] bg-white">
+      <div className="flex min-w-[1280px] items-center justify-end">
+        <ul className="flex items-center gap-[20px]">
+          <li className="relative flex items-center gap-[20px]">
+            <Link
+              to="/Join"
```

Comment:
```suggestion
              to="/join"
```
url은 소문자로 통일하는게 좋을 것 같습니다
밑에 로그인도 수정해주세요

Reply:
수정해서 다시 올렸습니다!! 확인 부탁드립니다